onnxruntime/wasm/api.h

```diff
@@ -24,6 +24,14 @@ using ort_run_options_handle_t = OrtRunOptions*;
 struct OrtValue;
 using ort_tensor_handle_t = OrtValue*;
 
+#ifdef ENABLE_TRAINING_APIS
+struct OrtTrainingSession;
+using ort_training_session_handle_t = OrtTrainingSession*;
+
+struct OrtCheckpointState;
+using ort_training_checkpoint_handle_t = OrtCheckpointState*;
```

Comment:
Do we need this alias? Can we just use OrtCheckpointState* ? In the current code I see use of both, at the very least we should stick to just 1 of the two.

Reply:
I think the aliases aren't needed, but since Training uses handles from Inference (run options handle, session options handle, etc) in parameters and return values, I stuck with the aliases & updated the code to more consistently use the aliases.